hack/bootstrap-cluster.sh

```diff
@@ -49,6 +49,13 @@ echo "Setting secrets for Tekton Results"
 if ! kubectl get namespace tekton-pipelines &>/dev/null; then
   kubectl create namespace tekton-pipelines
 fi
+
+OPENSSLDIR=`/usr/bin/openssl version -a |grep OPENSSLDIR |awk '{print $2}' |tr -d '"'`
+# make sure something didn't go wrong and a directory comes back
+if [ ! -d $OPENSSLDIR ]; then
```

Comment:
does not work, `$OPENSSLDIR` must be `"$OPENSSLDIR"`. In this case I would do -> `${OPENSSLDIR:-/etc/pki/tls}` in `openssl req` comand